docs/markdown/Using Pants/validating-dependencies.md

```diff
@@ -262,7 +266,9 @@ The previous example, using this alternative syntax for the selectors, would loo
 Glob syntax
 -----------
 
-The visibility rules support a basic glob syntax, using `*` as a match anything non-recursively with the `**` variant being for a recursive match anything. All rules are matched until the end of the path it is being applied to, so if there is not trailing wildcard (`*` or `**`) the end of the rule glob will match the end of the path. This allows for matching on file types/names regardless of where in the project tree they are:
+The visibility rules is all about matching globs. There are two primary kinds of globs supported, the simple glob syntax and the path glob syntax. Simple globs only supports `*` as a match anything and is otherwise case sensitive and are used for all glob values except for the `path` part of a __target rule spec__. The path glob syntax is described further below.
```

Comment:
This will seem pretty complicated to the user. Why not have one kind of glob, where `*` matches any character except `/` and `**` matches any character including `/` (which, IIUC, is what they mean in the current code).  Since non-paths won't have `/` in them, you can just use either, up to you.

Or do we really need two kinds of glob, for reasons I don't see?

Reply:
No, you're right. As I'm trying to consolidate the two we're close enough now to start treating them as the same documentation wise (and soon also implementation wise).